### PR TITLE
Editor: Confirmation: Shorting the heading text and make it easier to read.

### PR DESCRIPTION
**Pull request by shaunandrews**
The text on the publish confirmation is really long, and due to it's low contrast its hard to read. This PR increases the contrast and font size, as well and reducing the text down to a sentence.

Also, this removes the need for all the variations in the wording by avoiding using terms like "post" or "page." This means a lot less code, too.

Before:
screen shot 2018-03-05 at 3 21 30 pm

After:
screen shot 2018-03-05 at 3 57 58 pm

### CODE REVIEW

**Comment by mikedang** [comment]
Looking good! I'd tweak the text to "Almost there! Double-check your settings below, and use the big green button to publish!" to make it clear which settings we're referring to.

**Comment by drw158** [comment]
design looks good to me from the screenshots 👍

**Comment by michaeldcain** [comment]
Toggled the labels to re-run the tests.

File: client/post-editor/editor-confirmation-sidebar/header.jsx [hunk 1]
```diff
@@ -23,135 +23,47 @@ class EditorConfirmationSidebarHeader extends PureComponent {
...
+							strong: <strong />,
+						},
+					}
+				) }
+			</div>
+		);
 	}
 
 	renderScheduleHeader() {
-		const { post, postTypeLabel, translate } = this.props;
```
Comment (michaeldcain) [inline client/post-editor/editor-confirmation-sidebar/header.jsx]: With the removal of these, we can remove all of `connect` block and selectors.

**Pull request merged**